style.css

```diff
@@ -535,10 +592,9 @@ ul.points {
   display: flex;
   list-style-type: none;
   gap: 15px;
-  position: relative;
   right: 35px;
   width: 190px;
```

Comment:
Kindly avoid using fixed widths to have more responsiveness, try using rem/em values